src/lib/svelte-query/globalQueryClient.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * This is temporary until everything is moved to using svelte-query
```

Comment:
Why is this temporary?

Reply:
I am hoping we can pass this from components when calling various methods.